packages/component/src/Styles/createStyleSet.js

```diff
@@ -27,7 +27,7 @@ import createSuggestedActionsStyleSet from './StyleSet/SuggestedActionsStyleSet'
 import createSuggestedActionStyle from './StyleSet/SuggestedAction';
 import createTextContentStyle from './StyleSet/TextContent';
 import createTimestampStyle from './StyleSet/Timestamp';
-import createTypingActivityStyle from './StyleSet/TypingActivity';
+import createTypingIndicatorStyle from './StyleSet/TypingIndicator';
```

Comment:
Sorting.

Looks like Prettier isn't running on your box. 😄

Not entirely your fault, we should run Prettier as a bot on GitHub.